lang/en.lyaml

```diff
@@ -1418,3 +1425,52 @@ en:
         portalMissingScope: "Your account does not have access to this action. Talk to an account admin to request it."
         userMissingScope: "You don't have access to this action. Ask an account admin to change your permissions in Users & Teams settings."
         genericMissingScope: "Your access key does not allow this action. Please generate a new access key by running `hs auth personalaccesskey`."
+    doctor:
+      runningDiagnostics: "Running diagnostics..."
+      diagnosticsComplete: "Diagnostics complete"
+      accountChecks:
+        active: "Default account active"
+        inactive: "Default account isn't active"
+        inactiveSecondary: "Run {{ command }} to remove inactive accounts from your CLI config"
+        unableToDetermine: "Unable to determine if the portal is active"
+        pak:
+          invalid: "Personal access key is invalid"
+          valid: "Personal Access Key valid. {{ link }}"
+          viewScopes: "View selected scopes"
+      nodeChecks:
+        unableToDetermine: "Unable to determine what version of node is installed"
+        minimumNotMet: "Minimum Node version is not met, {{ nodeVersion }}"
+        success: "node v{{ nodeVersion }} is installed"
+      npmChecks:
+        notInstalled: "npm is not installed"
+        installed: "npm v{{ npmVersion }} is installed"
+        unableToDetermine: "Unable to determine if npm is installed"
+      projectDependenciesChecks:
+        missingDependencies: "missing dependencies in {{#cyan}}{{ dir }}{{/cyan}}"
+        missingDependenciesSecondary: "Run {{ command }} to install all project dependencies locally"
+        unableToDetermine: "Unable to determine if dependencies are installed {{ dir }}"
+        success: "App dependencies are installed an up to date"
+      files:
+        invalidJson: "invalid JSON in {{#cyan}}{{ filename }}{{/cyan}}"
```

Comment:
iirc we don't typically use cyan for filepaths. We use bold. Was this from the mocks?

Reply:
I just messaged Sandra to confirm

Reply:
Changed them to bold